Tests/CommandsTests/BuildCommandTests.swift

```diff
@@ -880,13 +891,20 @@ class BuildCommandSwiftBuildTests: BuildCommandTestCases {
     override func testAtMainSupport() async throws {
         #if !os(macOS)
         throw XCTSkip("SWBINTTODO: File not found or missing libclang errors on non-macOS platforms. This needs to be investigated")
-        #endif
-
+        #else
         try await super.testAtMainSupport()
+        #endif
     }
 
     override func testAutomaticParseableInterfacesWithLibraryEvolution() async throws {
-        throw XCTSkip("SWBINTTODO: The test fails because when the unsafe flag for a target is set to '-enable-library-evolution' it is not producing the correct .swiftinterface files. This needs to be investigated")
+        try await fixture(name: "Miscellaneous/LibraryEvolution") { fixturePath in
```

Comment:
question: is this different than the parent?  if so, can we simply call  `super.test...()`?